etc/rollup/config.js

```diff
@@ -32,11 +32,16 @@ export default {
   external: [...Object.keys(pkg.dependencies || {}), ...Object.keys(pkg.peerDependencies || {})],
   plugins: [
     customResolveOptions({ extensions }),
-    babel({
-      presets: [['react-app', { flow: false, typescript: true, absoluteRuntime: false }]],
-      babelHelpers: 'runtime',
-      extensions,
-      exclude: 'node_modules',
+    styles(),
+    swc({
+      jsc: {
+        jsc: {
```

Comment:
Are you sure this is correct? `jsc` inside `jsc` looks a bit strange.

Reply:
Fix it. Not sure how it was working before 🤔